components/definitions/newTab.d.ts

```diff
@@ -143,6 +144,7 @@ declare namespace NewTab {
   }
 
   export interface BinanceWidgetState {
+    binanceSupported?: boolean
```

Comment:
@ryanml is this meant to be here? I think this got moved up to the root state. Or actually I think there's a bug.

In `NEW_TAB_SET_INITIAL_DATA` the value is set on `newTabData.binanceReported`,
and in all the react components, it's read from this same property. But both Storbyook and `binance_reducer.ts` attempt to set it to `newTabData.binanceState.binanceReported`. I don't think anything reads it from there, is that right?